app/js/contentscript.js

```diff
@@ -10,36 +11,58 @@ class Provider {
       chrome.runtime.sendMessage({ bg: 'isActive' }, response => {
         this.listen()
         if (!response.isActive) {
-          console.warn('paster is disable. Please click Turn ON button in popup window.') // eslint-disable-line no-console
+          this.setState({
+            status: 'OFF',
+            message: 'Please click turn on button in popup window'
+          })
+          this.renderBadge('OFF')
           return
         }
         this.readyState()
       })
     })
   }
+  setState (data: object): void {
+    if (data) {
+      state = data
+      return
+    }
+    state = {}
```

Comment:
IMO, you don't need clear state here.
If you want to clear state, you may provide `clear()` method.